dev/VSIX/Extension/Cpp/Component/source.extension.vsixmanifest

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.WindowsAppSDK.Cpp" Version="|%CurrentProject%;GetVSIXVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>Windows App SDK (C++)</DisplayName>
+    <Description xml:space="preserve">The Microsoft Windows App SDK Visual Studio extension adds C++ project and item templates to support building Windows apps and components.</Description>
+    <MoreInfo>https://github.com/microsoft/WindowsAppSDK/</MoreInfo>
+    <License>LICENSE</License>
+    <GettingStartedGuide>https://github.com/microsoft/WindowsAppSDK/</GettingStartedGuide>
+    <ReleaseNotes>https://github.com/microsoft/WindowsAppSDK/releases</ReleaseNotes>
+    <Icon>WindowsAppSDK.png</Icon>
+    <PreviewImage>WindowsAppSDK.png</PreviewImage>
+    <Tags>WindowsAppSDK, WinUI, XAML, C++, C++/WinRT</Tags>
+    <Preview>false</Preview>
+  </Metadata>
+  <Installation AllUsers="true">
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+  </Installation>
+  <Prerequisites>
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,18.0)" DisplayName="Visual Studio core editor" />
```

Comment:
>Microsoft.VisualStudio.Component.CoreEditor [](http://example.com/codeflow?start=22&length=43)

I thought component VSIXs aren't allowed to take a dependency on CoreEditor unless they're part of a core component?

Reply:
Correct, and good catch.

I was attempting to do this in two separate commits, but in all my attempts to try and fix the MSBuild weirdness, I started getting a little scope creep. I'll just finish this off and complete both the language split and component splits right now.